### ISSUE TITLE
Deviation in reproduction

### ISSUE THREAD

**Issue by lemonsis** [rootnx/UniCoRN#2]
Hi, authors. Thanks for the code and effort. I'm trying to reproduce the results in the paper with one A100-40G gpu. Without changing any hyper-parameters in the given code but batch size, I get the following results for random split with series length 10, which are far below the results in the paper. One weird thing during reproduction happened in line147 and 148 of FMRI_datasets.py, I got count=440 instead of 280 in comment. I think this might cause the different score, but the gap is still too large.

test_my_BLEU_1: 0.5045886462272847
test_my_BLEU_2: 0.31371376122208494
test_my_BLEU_3: 0.21931830490050072
test_my_BLEU_4: 0.16312840950077365
test_rouge-1_r: 0.4198069938456754
test_rouge-1_p: 0.49354893783577375
test_rouge-1_f: 0.45229051420776223
test_rouge-2_r: 0.1586514126178775
test_rouge-2_p: 0.16305219008892882
test_rouge-2_f: 0.1606447368412828
test_rouge-l_r: 0.37124970964331544
test_rouge-l_p: 0.43491279504550184
test_rouge-l_f: 0.39933078873930994

**Comment by lemonsis** [comment]
I fixed this issue. Changing the learning rate in phase 3 from 1e-3 to 1e-4 and the final learning rate in phase 3 from 1e-5 to 1e-6 will lead to the results in the paper for me.
